oem/ibm/configurations/pdr/ibm,bonnell/4.json

```diff
@@ -0,0 +1,423 @@
+{
+ "sensorPDRs": [
+   {
+    "pdrType" : 4,
+    "entries" : [{
+        "type" : 45,
+        "instance" : 1,
+        "container" : 1,
+        "sensors" : [{
+            "set" : {
+                "id" : 17,
+                "size" : 1,
+                "states" : [1,2]
+            },
+            "dbus" : {
```

Comment:
this should be like below
            "dbus" : {
                "path": "/xyz/openbmc_project/led/physical/virtual_enc_id",
                "interface": "xyz.openbmc_project.Led.Physical",
                "property_name": "State",
                "property_type": "string",
                "property_values": [
                         "xyz.openbmc_project.Led.Physical.Action.Off",
                         "xyz.openbmc_project.Led.Physical.Action.On || xyz.openbmc_project.Led.Physical.Action.Blink"
                ]
             }